packages/admin-next/dashboard/src/routes/orders/order-list/const.ts

```diff
@@ -4,7 +4,8 @@ const DEFAULT_PROPERTIES = [
   "created_at",
   "email",
   "display_id",
-  // "payment_status", // -> TODO replacement for this
+  "payment_status",
```

Comment:
I think we have this commented out in the order details as well, can you also check it's working as expected there?

Reply:
there was a TODO which is no longer needed, but currently, we don't show the payment section on the details page since it needs to be revisited